src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import core.basesyntax.service.ColorSupplier;
+
+public abstract class Figure implements CalculatorArea, Drawable {
+    private String color = new ColorSupplier().getRandomColor();
```

Comment:
```suggestion
    private String color;
```
I wrote that getting a random color should be done in FigureSupplier